Frontend/src/User/UserPlacesList.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import { selectUpdatePlace } from '../Redux/updatePlaceSlice';
+import UserPlaceItem from './UserPlaceItem';
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+
+const UserPlacesList = () => {
+  const updatePlaceData = useSelector(selectUpdatePlace);
+  const user = sessionStorage.getItem('user');
+  let userPlaces;
+  useEffect(() => {
+    axios.get(`http://localhost:8000/memo_places/user=${user.id}`).then((data) => {
```

Comment:
Tutaj to już wg nie tak jak powinno być.
 `http://localhost:8000/memo_places/places/user=${user.id}` tak to powinno wyglądać